lib/beaker/dsl/install_utils.rb

```diff
@@ -1453,6 +1453,100 @@ def build_ignore_list(opts = {})
         ignore_list << '..' unless ignore_list.include? '..'
         ignore_list
       end
+
+      # Fix for PUP-2365
+      def install_forge_cert
+        hosts.each do |host|
+          if host['platform'] =~ /windows/
+            configure_puppet_on(host,opts)
+          end
+        end
+      end
+
+      # This method is a fix for PUP-2365 when a windows agent cannot download
```

Comment:
Thus is pretty permanent because even when PUP-2365 is fixed, there will be older versions of puppet that users may want to install that do not include that fix. Although when the issue is resolved and released it would be useful to come back and update the comment.